Web/Design-Pattern-2/Assignment-3/src/Test.java

```diff
@@ -0,0 +1,19 @@
+
+public class Test  extends Game
```

Comment:
ACTION: Class name should be different from "Test" as "Test" is a reserved keyword.